lighthouse-core/report/scripts/lighthouse-report.js

```diff
@@ -19,40 +19,9 @@
 
 'use strict';
 
-/* Using ES5 to broaden support */
-function LighthouseReport() {
-  this.printButton = document.querySelector('.js-print');
-  this.viewUserFeature = document.querySelector('.js-report-by-user-feature');
-  this.viewTechnology = document.querySelector('.js-report-by-technology');
-  this.reportItems = document.querySelectorAll('.report-section__item');
-  this.helpToggles = document.querySelectorAll('.subitem__help-toggle');
-
-  this.toggleHelpText = this.toggleHelpText.bind(this);
-
-  this.addEventListeners();
-}
-
-LighthouseReport.prototype = {
-
-  onPrint: function() {
-    window.print();
-  },
-
-  toggleHelpText: function(e) {
-    const item = e.currentTarget.closest('.subitem');
-    item.classList.toggle('--show-help');
-  },
-
-  addEventListeners: function() {
-    this.printButton.addEventListener('click', this.onPrint);
-    [...this.helpToggles].forEach(node => {
-      node.addEventListener('click', this.toggleHelpText);
-    });
-  }
-};
-
 window.addEventListener('DOMContentLoaded', _ => {
-  // TODO: remove these two comments when we upgrade to latest eslint
-  // eslint-disable-next-line no-new
-  new LighthouseReport();
+  var printButton = document.querySelector('.js-print');
```

Comment:
Originally we had this as es5 so it could support more browsers, but then we added spread and custom properties and whatever else, so I guess might as well do `const` :)